pkg/defaults/defaults.go

```diff
@@ -544,6 +544,9 @@ const (
 
 	// EnableK8sNetworkPolicy enables support for K8s NetworkPolicy.
 	EnableK8sNetworkPolicy = true
+
+	// PolicyCIDRSelectsNodes enables support for selecting nodes via CIDR selectors
+	PolicyCIDRSelectsNodes = true
```

Comment:
I know this commit already says DNM (do not merge), but just highlighting that the default here should be false.

Reply:
Yup; in the commit that introduces this, the default is false. I will drop the DNM commit approximately when this PR is ready to merge.

Just trying to get CI signal for now.

Reply:
I've removed this as default.